students/JeroenSweerts/Lesson3/mailroom.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+donors = ['papa', 'mama', 'bompa', 'bobonne', 'onbekende']
+donations = [[100, 5, 15], [12, 200, 2, 66], [1000], [500, 500], [1000000]]
```

Comment:
Can you think of a way to combine these into a single data structure? Then you wouldn't need to keep the indexes of the two lists in sync and this would simplify some of your lookup code below.